src/OpenTelemetry.Resources.ProcessRuntime/ProcessRuntimeDetector.cs

```diff
@@ -19,39 +20,32 @@ internal sealed class ProcessRuntimeDetector : IResourceDetector
     /// <returns>Resource with key-value pairs of resource attributes.</returns>
     public Resource Detect()
     {
-        var frameworkDescription = RuntimeInformation.FrameworkDescription;
-        string netRuntimeName;
+        string netRuntimeVersion = string.Empty;
 #if NETFRAMEWORK
         var netFrameworkVersion = GetNetFrameworkVersionFromRegistry();
-#endif
-
-        var lastSpace = frameworkDescription.LastIndexOf(' ');
-        if (lastSpace != -1)
+#pragma warning disable CS8602 // Dereference of a possibly null reference.
```

Comment:
It's better to use `!` as the less verbose built-in language feature to do this if the code can't be refactored to not produce false-positives.